http.go

```diff
@@ -221,6 +234,8 @@ func parseArgs(fset *flag.FlagSet, args []string) (*params, error) {
 
 	fset.BoolVar(&p.insecure, "insecure", false, "skip HTTPS certificate checking")
 
+	fset.BoolVar(&p.checkStatus, "check-status", false, "exit with an error if the HTTP status si one of 3xx, 4xx or 5xx the exit status will be 3, 4 or 5 respectively")
```

Comment:
fset.BoolVar(&p.checkStatus, "check-status", false, "if the HTTP status is not 2xx, print a warning and use the first digit of the status code as exit status")

?